lib/actions.js

```diff
@@ -84,8 +84,15 @@ exports.ensureEmptyDirectory = function() {
   if(fs.existsSync(this.destinationRoot())) {
     // A valid directory won't have any files/folders.
     var files = fs.readdirSync(this.destinationRoot());
+    var file;
     if (files.length > 0) {
-      this.env.error(chalk.red(this.destinationRoot(), 'is not an empty directory'));
+      var count;
```

Comment:
OK. This would not be a problem if the files were written with the memory filesystem, since writes are deferred. This goes with my comment regarding the use of `fs` rather than `this.fs`.